vue/components/ui/molecules/table/table.vue

```diff
@@ -519,6 +523,7 @@ export const Table = {
         },
         rowClasses(item) {
             const base = {
+                checked: this.checkedItemsData[item.id],

```

Comment:
With this, this behaviour will be enabled, so `rowSelection` is not used. Have you checked if the `selected` style and the logic of `selectedRowData` is actually being used here? Because it seems that logic was abandoned, since `selectedIdData` is not being used anywhere and `selectedRow` is not being passed by filter. Perhaps these can be recycled.
If not, we can add a new variable similar to `rowSelection` but for checked

Reply:
I think `rowSelection` is for one selected row only (so, the last selected/checked row). Visually they amount to the same, because the styles do not differ here, but they could (so having different classes may be a good thing)

Reply:
Selected logic is working, it works as the row that was clicked, even if outside the checkbox. Also, it amounts to the last row clicked. So there can be many checked, but only one marked as selected.

Reply:
I added the `rowSelection` to propagate to the table component, meaning before was not passed, meaning it is not used anywhere for sure. So perhaps we could remove this logic. What do you think?

Reply:
I would not remove it, since the behaviour is already there. I would pass props to allow it, like the `rowSelection` and `selectedRow`. But similar to this behaviour, a new prop should be passed to allow checked selection style.

Reply:
One thing I would check is if `selectedIdData` is really being used anywhere or if it can be removed